src/style.css

```diff
@@ -1,3 +1,108 @@
+/* Reset de margens padrão do navegador */
+body, h1, h2, h3, h4, h5, h6, p, ul, ol, li, figure, figcaption, blockquote, dl, dd {
+  margin: 0;
+}
+
+/* Importação da fonte Roboto */
 body {
+  background-color: rgb(230, 227, 227);
+  font-family: 'Roboto', sans-serif;
+  font-weight: bold;
+  display: flex;
+  justify-content: center; /* Centraliza horizontalmente */
+  align-items: center; /* Centraliza verticalmente */
+}
+
+/* Estilo do header */
+header {
+  width: 1200px;
+  height: 60px;
```

Comment:
Height should be set in the nav item not in header [Reference](https://github.com/mate-academy/layout_moyo-header#:~:text=the%20height%20should%20be%20set%20for%20nav%20links%20(not%20the%20header)%2C%20take%20it%20from%20the%20design.)